scripts/db/postgres/dev/upgrade_dev_database.sh

```diff
@@ -4,17 +4,16 @@ set -o pipefail
 DEPLOYMENT_CODE="$1"
 DB_NAME="$2"
 SCRIPT_DIR=$(cd "$(dirname "$0")" ; pwd -P)
-POSTGRES_SYSTEM_USER="$("${SCRIPT_DIR}/get_postgres_system_user.sh")"
+POSTGRES_SYSTEM_USER="postgres"
 ZEN_SRC_ROOT=$(git rev-parse --show-toplevel)
 
+
 # Test to see if the destination database already exists.
-DB_EXISTS_TEST=$(sudo -u "${POSTGRES_SYSTEM_USER}" \
-  psql postgres -tAc "SELECT 1 FROM pg_database WHERE datname='${DB_NAME}'")
+DB_EXISTS_TEST=$(psql -h ${DATABASE_HOST:-localhost} -U postgres postgres -tAc "SELECT 1 FROM pg_database WHERE datname='${DB_NAME}'")
```

Comment:
Hmm... as above - why don't I just go for no fallback at all? I'm going to go for `${DATABASE_HOST}` instead of `${DATABASE_HOST:-localhost}`

Reply:
Yeah a refactor here to be able to just use the DATABASE_URL like `psql DATABASE_URL` would be ideal. It would need to be able to handle the initial case where the table hasn't been created yet so the full DATABASE_URL would fail. I think that would be worth it since `DATABASE_HOST` just exists in this file